kg_microbe/transform_utils/bacdive/bacdive.py

```diff
@@ -348,14 +348,7 @@ def _process_metabolites(self, dictionary, ncbitaxon_id, key, node_writer, edge_
                                 metabolite_id,
                                 None,
                                 BACDIVE_PREFIX + key,
-                            ],
-                            [
-                                medium_id,
-                                MEDIUM_TO_METABOLITE_EDGE,
```

Comment:
this was the edge to the antibitioc -- but that medium should have this edge because this would be medium:X+ antibitioc, really there should be an assay node here but can fix in the future.